content/applications/finance/payment_providers/adyen.rst

```diff
@@ -106,18 +105,12 @@ URLs, proceed as follows:
      :alt: Copy the prefix for the Adyen APIs
 
 #. In Odoo, :ref:`navigate to the payment provider Adyen <payment_providers/add_new>`.
-#. In the :guilabel:`Checkout API URL` field, enter the following URL and replace `yourprefix` with
-   the prefix you previously saved:
-   `https://yourprefix-checkout-live.adyenpayments.com/checkout`
-#. In the :guilabel:`Recurring API URL` field, enter the following URL and replace `yourprefix` with
-   the prefix you previously saved:
-   `https://yourprefix-pal-live.adyenpayments.com/pal/servlet/Recurring`.
+#. In the :guilabel:`API URL Prefix` field, enter the prefix you previously saved.
 
 .. note::
-   If you are trying Adyen as a test, you can use the following URLs instead:
+   If you are trying Adyen as a test, you can use the following Prefix instead:
 
-   - :guilabel:`Checkout API URL`: `https://checkout-test.adyen.com`
-   - :guilabel:`Recurring API URL`: `https://pal-test.adyen.com/pal/servlet/Recurring`
+   - :guilabel:`API URL Prefix`: `checkout-test`
```

Comment:
Since there is only one prefix, I would remove the bullet list and merge it with the previous sentence as follows:
If you are trying Adyen as a test, you can use the following :guilabel:`API URL Prefix` instead: `checkout-test`.
What do you think?

Reply:
Yeah, sounds all right.